src/main/java/com/project/springbootproject/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package com.project.springbootproject.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
1. Why to make checked exception to escalate it whole way up to controller without handling?
Make it unchecked and avoid overdeclaration on methods/interfaces/etc
2. "Consider to handle the RegistrationException using the GlobalExceptionHandler"
cannot see this point of task